e2e/volume_ops_test.go

```diff
@@ -271,6 +272,39 @@ func testVolumeDelete(t *testing.T) {
 func testVolumeStart(t *testing.T) {
 	r := require.New(t)
 	r.Nil(client.VolumeStart(volname, false), "volume start failed")
+
+	vol, err := client.BricksStatus(volname)
+	for index := range vol {
+		r.NotZero(vol[index].Port)
+		r.True(vol[index].Online)
+	}
+
+	vol, err = client.BricksStatus(volname)
+	killBrick := vol[0].Info.Path
+	process, err := os.FindProcess(vol[0].Pid)
+	r.Nil(err, fmt.Sprintf("failed to find brick pid: %s", err))
+	err = process.Signal(syscall.Signal(15))
+	r.Nil(err, fmt.Sprintf("failed to kill brick: %s", err))
+
+	time.Sleep(3 * time.Second)
+
+	vol, err = client.BricksStatus(volname)
+	for index := range vol {
+		if vol[index].Info.Path == killBrick {
+			r.Zero(vol[index].Port)
+		}
+	}
+
+	r.Nil(client.VolumeStart(volname, true), "volume force start failed")
+
```

Comment:
You may need a sleep here. The brick that was forcefully started has to send a SIGNIN request. Only after that will glusterd2 know it's port.

Reply:
Done